### DISCUSSION TITLE
import pytorch_lightning as pl does not work on colab

### DISCUSSION THREAD

**Discussion by Bajo1994** [Lightning-AI/pytorch-lightning#6425] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I can install pl in colab by
`!pip install pytorch-lightning==1.2.2 --quiet`
but I cannot import it by
`import pytorch_lightning as pl`
I am thankful if you help me with this issue.

**Comment by NoraAlt** [comment]
Solution is here: 
https://github.com/PyTorchLightning/pytorch-lightning/issues/6426

**Comment by SkafteNicki** [comment, marked as answer]
Please upgrade to version 1.2.3 (released yesterday) where this issue was solved.
